itest/test/spectre/metrics_test.py

```diff
@@ -0,0 +1,201 @@
+import json
+import time
+from collections import namedtuple
+
+import pytest
+
+from util import get_through_spectre
+
+METRICS_FILE = '/var/log/metrics/metrics.log'
+
+Metric = namedtuple('Metric', ['dimensions', 'value', 'type'])
+
+
+@pytest.fixture
+def log_file():
+    with open(METRICS_FILE, 'r') as fp:
+        # Ignore all the lines logged before the start of the test
+        fp.readlines()
+        yield fp
+
+
+def _parse_dimensions(dims_str):
+    # meteorite dimensions are encoded as a list of list
+    # let's convert them back to a map so they're easier to check
+    dims_map = {}
+    dims_list = json.loads(dims_str)
+    for el in dims_list:
+        dims_map[el[0]] = el[1]
+
+    return dims_map
+
+
+def _load_metrics(log_file):
+    metrics = []
+    time.sleep(0.5)
```

Comment:
Is the test flaky without this, or straight up failing?

Reply:
straight up failing